drgn_tools/scsi.py

```diff
@@ -40,13 +42,45 @@ def for_each_scsi_host(prog: Program) -> Iterator[Object]:
             yield container_of(dev, "struct Scsi_Host", "shost_dev")
 
 
+def host_module_name(shost: Object) -> str:
+    """
+    Fetch the module name associated with the scsi host.
+    returns: the module name string.
+    """
+    try:
+        name = shost.hostt.module.name.string_().decode()
+    except ValueError:
+        name = escape_ascii_string("unknonw")
```

Comment:
I don't think "escape_ascii_string" is required here, you are passing in a str not a bytes

Reply:
it is just null string check, I’ll change to drgn.FaultError